### ISSUE TITLE
Results Not Good When Reproducing Pretraining on Simpler Dataset

### ISSUE THREAD

**Issue by chang-github-00** [LatentActionPretraining/LAPA#31]
Dear authors, 

Thank you for the amazing work and open-sourcing. We are following your work and hope to reproduce pretraining performances.

However, we find that 

1. The performances after pretraining is only slightly better than directly fine-tuning the base model `LargeWorldModel/LWM-Chat-1M-Jax`
2. Our reproduced pretraining seems to have negative effects on Simpler performances, and is more than 10% worse in terms of accuracy.

We strictly follow all the training configurations. The results and training curves are here: https://github.com/chang-github-00/LAPA-Reproduce

Wonder if you could help troubleshoot the bad performances ?

Thanks,
Chang

**Comment by seonghyeonye** [comment]
There might be a mismatch between our evaluation protocol with yours. For direct fine-tuning of the base model (scratch), we got 34.4. Is the score that you reported (60.42) is not the grasp rate or move rate (instead of the final success rate)?
